### ISSUE TITLE
how to change learning rate and epochs for training in your config file?

### ISSUE THREAD

**Issue by Jamestrump** [open-mmlab/mmdetection#5869]

**Comment by RangiLyu** [comment]
You can refer to the doc of config file https://mmdetection.readthedocs.io/en/latest/tutorials/config.html
